tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -66,6 +66,7 @@ Verify A Default Image Is Provided And Starts Successfully
     @{user_data} =    Get Previously Selected Notebook Image Details
     Launch JupyterHub Spawner From Dashboard
     Spawn Notebook
+    Sleep  7
```

Comment:
This should be `Wait for JupyterLab Splash Screen  timeout=30`

Reply:
Updated.